charts/posit-chronicle/ci/complex-values.yaml

```diff
@@ -0,0 +1,65 @@
+image:
+  tag: "latest"
+  imagePullPolicy: "IfNotPresent"
+
+serviceaccount:
+  create: true
+  annotations:
+    eks.amazonaws.com/role-arn: arn:aws:iam::123123123123123:role/chronicle-serviceaccount-role
+
+service:
+  # -- The port to use for the REST service
+  port: 12345
+  annotations:
+    test-annotation: test-service-annotation
+  labels:
+    test-label: test-service-label
+    test-label2: test-service-label2
+  selectorLabels:
+    test-selector-label: test-service-selectorLabel
+    test-selector-label2: test-service-selectorLabel2
+
+pod:
+  replicas: 4
+  command: "/bash"
+  args: ["echo", "hello world"]
+  # -- Optional environment variables
+  env:
+  - name: "TEST_ENV_VAR"
+    value: "test-value"
+  annotations:
+    test-annotation: test-service-annotation
+  labels:
+    test-label: test-service-label
+    test-label2: test-service-label2
+  selectorLabels:
+    test-selector-label: test-service-selectorLabel
+    test-selector-label2: test-service-selectorLabel2
+
+storage:
+  persistentVolumeSize: 10Gi
+
+config:
```

Comment:
nice callout. I noticed we don't do this in the [workbench-config section](https://github.com/rstudio/helm/blob/main/charts/rstudio-workbench/values.yaml#L371), but if ppm and connect do we might as well

Reply:
Workbench is a little bit different, but I would use Connect and Package Manager as my reference since they are newer products, and they are pretty consistent between each other in terms of naming conventions.